plugin-server/src/worker/ingestion/person-state.ts

```diff
@@ -25,11 +25,13 @@ export const mergeFinalFailuresCounter = new Counter({
 export const mergeTxnAttemptCounter = new Counter({
     name: 'person_merge_txn_attempt_total',
     help: 'Number of person merge attempts.',
+    labelNames: ['call', 'oldPersonIdentified', 'newPersonIdentified', 'poEEmbraceJoin'],
```

Comment:
While VictoriaMetric handles high cardinality a bit better than Prometheus, personId cardinality will most probably overwhelm VictoriaMetric.
We can definietely add 'call' and 'poEEmbraceJoin', but for 'oldPersonIdentified' and 'newPersonIdentified' we should look into structured logging instead

Reply:
`oldPersonIdentified` and `newPersonIdentified` are just `boolean`

Reply:
oh, my bad!